doc/bc/changes-8.0.md

```diff
@@ -23,6 +23,11 @@ Changes affecting version compatibility with former or future versions.
   eZ Platform REST Bundle available via
   [ezsystems/ezplatform-rest](https://github.com/ezsystems/ezplatform-rest) package.
 
+* Deprecated method `getName` from the interface `eZ\Publish\SPI\FieldType\FieldType` has been changed. 
+  Now it accepts two additional parameters: `FieldDefinition $fieldDefinition` and `string $languageCode`
+  
+* Interface `eZ\Publish\SPI\FieldType\FieldType` has been transformed to abstract class.   
+
```

Comment:
I'm missing information that `Nameable` has been dropped.

Reply:
@alongosz done